testing/trino-testing/src/main/java/io/trino/testing/sql/TestTable.java

```diff
@@ -35,35 +35,49 @@
 
```

Comment:
It might make more sense to introduce a new TestTable within the trino-druid module. After all other modules shouldn't be affected by Druid's limitations.

Please extract to separate class.

Reply:
The consideration was there might be other databases that don't support creating tables and inserting data -- but you are right, this might be a premature abstraction. WIll modify.